wolfhsm/wh_client_crypto.h

```diff
@@ -115,27 +115,25 @@ int wh_Client_ImportCurve25519Key(whClientContext* ctx, curve25519_key* key,
  * @return int Returns 0 on success or a negative error code on failure.
  */
 
-int wh_Client_ExportCurve25519Key(whClientContext* ctx, whKeyId keyId,
-        curve25519_key* key, uint32_t label_len, uint8_t* label);
-
-/* TODO: Make a Curve25519 key on the server based on the flags and update the local
- * struct accordingly */
-int wh_Client_MakeCurve25519Key(whClientContext* ctx,
-        uint32_t size,
-        whKeyId *inout_key_id, whNvmFlags flags,
-        uint32_t label_len, uint8_t* label,
-        curve25519_key* key);
+int wh_Client_Curve25519ExportKey(whClientContext* ctx, whKeyId keyId,
+        curve25519_key* key, uint16_t label_len, uint8_t* label);
 
 /* TODO: Generate a Curve25519 key on the server and put it in the server keycache */
```

Comment:
all these functions are no longer a TODO, and also need doxygen API docs

Reply:
The TODO is for me to add the doxygen!  :)